File: src/targets/java/asynchttp/fixtures/application-form-encoded.java
AsyncHttpClient client = new DefaultAsyncHttpClient();
client.prepare("POST", "http://mockbin.com/har")
  .setHeader("content-type", "application/x-www-form-urlencoded")
  .setBody("foo=bar&hello=world")
  .execute()
  .toCompletableFuture()
  .thenAccept(System.out::println)
  .join();

client.close();

File: src/targets/javascript/xhr/fixtures/application-form-encoded.js
const data = 'foo=bar&hello=world';

const xhr = new XMLHttpRequest();
xhr.withCredentials = true;

xhr.addEventListener('readystatechange', function () {
  if (this.readyState === this.DONE) {
    console.log(this.responseText);
  }
});

xhr.open('POST', 'http://mockbin.com/har');
xhr.setRequestHeader('content-type', 'application/x-www-form-urlencoded');

xhr.send(data);